osidb/tests/test_package_versions.py

```diff
@@ -11,30 +11,29 @@ class TestPackageVersions(object):
     def test_create_cve_v5_version(self):
         """test raw verison range creation"""
 
-        vr_1 = CVEv5Version.objects.create(
-            version="3.2.1",
+        flaw = FlawFactory()
+        pkg = PackageFactory(
+            acl_read=flaw.acl_read,
+            acl_write=flaw.acl_write,
+            flaw=flaw,  # make _validate_acl_identical_to_parent_flaw happy
         )
 
-        with pytest.raises(ValidationError):
-            vr_1.validate()
-
-        vr_2 = CVEv5Version.objects.create(
+        PackageVer.objects.create(
             version="3.2.1",
-            status=VersionStatus.UNAFFECTED,
+            package=pkg,
         )
 
-        assert vr_2.validate() is None
-        assert vr_2.save() is None
+        assert pkg.validate() is None
+        assert pkg.save() is None
 
     def test_create_cve_v5_package_versions(self):
 
-        vrs_1 = CVEv5PackageVersions()
+        vrs_1 = Package()
         with pytest.raises(ValidationError):
             vrs_1.validate()
 
-        vr_1 = CVEv5VersionFactory()
-
-        vrs_2 = CVEv5PackageVersionsFactory.create(versions=(vr_1,))
+        vrs_2 = PackageFactory()
+        PackageVerFactory(package=vrs_2)
 
         assert vrs_2.validate() is None
         assert vrs_2.save() is None
```

Comment:
The original `test_create_cve_v5_version` can be removed as it tested `status`, which is no longer here. So we will end up with only one test, which could be moved to `test_flaw` (there is no need to have a separate file for it).

Reply:
Good point. I'd prefer to add a commit (that won't be squashed at merge) that cleans up the tests. The reason is that the old tests were valuable during the transition from old models to new models to prevent regressions. Some additional context also in comment https://github.com/RedHatProductSecurity/osidb/pull/307#discussion_r1320131359

Reply:
Resolved in rebased & partially squashed commits by adding explanatory commit messages into the three initial commits and deleting the file in the third commit. There's truly no need to test basic and obvious django functionality.